### ISSUE TITLE
"firebase init" fails if storage is selected without a default project

### ISSUE THREAD

**Issue by kroikie** [firebase/firebase-tools#4917]
### [REQUIRED] Environment info

**firebase-tools: 11.8.0**

**Platform: macOS**

### [REQUIRED] Test case

Initialize firebase selecting storage without a default project.

### [REQUIRED] Steps to reproduce

1. firebase init
2. select "storage"
3. select "Don't set up a default project"
4. Error!!!

### [REQUIRED] Expected behavior

Initialization should complete successfully. Maybe by skipping any calls to Cloud Storage if no default project option is selected.

### [REQUIRED] Actual behavior

Initialization fails with error:

```
Error: Cloud resource location is not set for this project but the operation you are attempting to perform in Cloud Storage requires it. Please see this documentation for more details: https://firebase.google.com/docs/projects/locations
```

**Comment by aalej** [comment]
I can't reproduce this issue using the latest version of firebase-tools(currently v13.29.0). From what I can tell, this was fixed in #7905. Since the cloud resource location requirement was removed, the API call that raises this error is no longer being made. The fix for this should be available on firebase-tools [v13.24.1](https://github.com/firebase/firebase-tools/releases/tag/v13.24.1).

If anyone is still encountering this issue using firebase-tools >= v13.24.1, please leave a comment so that we can re-open this. Thanks!

**Comment by alexander-rod** [comment]
I am also running into this issue.
